core/os/device/android.go

```diff
@@ -16,6 +16,8 @@ package device
 
 import "fmt"
 
+const AndroidMinimalSupportedAPIVersion = 23
```

Comment:
Do we have this constant sitting somewhere else as well and can we merge them?

Reply:
After chatting offline, such a constant does not appear to be existing.
Created #3043 to fix this up in due course.